src/SFA.DAS.ApprenticeCommitments/Application/Commands/CreateApprenticeship/CreateApprenticeshipCommandHandler.cs

```diff
@@ -57,17 +58,19 @@ public class CreateApprenticeshipCommandHandler : IRequestHandler<CreateApprenti
                 CommitmentsApprovedOn = command.CommitmentsApprovedOn,
             });
 
-            await _apprenticeLoginService.SendInvitation(new SendInvitationModel
+            // return parameters for the invitation
+            var res = new CreateApprenticeshipResponse
             {
+                ClientId = _loginConfiguration.IdentityServerClientId,
```

Comment:
Also be beneficial to delete the APIM config section related to this

Reply:
Created https://skillsfundingagency.atlassian.net/browse/CS-777 to move the configs.  regarding removing the old configs, SendInvitationRemindersCommandHandler also makes use of them so that would need consideration